core/src/components/cat-tooltip/cat-tooltip.tsx

```diff
@@ -0,0 +1,68 @@
+import {Component, h, Host, Prop} from '@stencil/core';
+import {autoUpdate, computePosition, flip, offset, Placement} from "@floating-ui/dom";
+
+@Component({
+  tag: 'cat-tooltip',
+  styleUrl: 'cat-tooltip.scss',
+  shadow: true,
+})
+export class CatTooltip {
+  private tooltip?: HTMLElement;
+  private trigger?: Element;
+
+  /**
+   * The content of the tooltip
+   */
+  @Prop() content!: string;
+
+  /**
+   * Specifies that the tooltip should be disabled. A disabled tooltip is unusable,
+   * and invisible. Corresponds with the native HTML disabled attribute.
+   */
+  @Prop() disabled = false;
+
+  /**
+   * The placement of the tooltip.
+   */
+  @Prop() placement: Placement = 'top';
+
+  componentDidLoad(): void {
+    if (this.trigger && this.tooltip) {
+      autoUpdate(this.trigger, this.tooltip, () => this.update());
+    }
+  }
+
+  render() {
+    return (
+      <Host>
+        <div
+          aria-valuetext={this.content}
+          role="tooltip"
```

Comment:
Looking at a couple of resources, the aria tooltip role is actually discouraged for tooltips :D. See: https://sarahmhigley.com/writing/tooltips-in-wcag-21/#:~:text=role%3D%22tooltip%22%3A%20the,with%20aria%2Ddescribedby. I think we should follow the approach of angular Material closely. Also with adding a hidden content: https://material.angular.io/components/tooltip/overview#accessibility